reporting/impl/src/main/resources/reports/templates/migration-issues.ftl

```diff
@@ -134,7 +134,11 @@
                 <div class="page-header page-header-no-border">
                     <h1>
                         <div class="main">${reportModel.reportName}
-                        <i class="glyphicon glyphicon-info-sign" data-toggle="tooltip" data-placement=right title="${reportModel.description}"></i></div>
+                        <i class="glyphicon glyphicon-info-sign" data-toggle="tooltip" data-placement=right title="${reportModel.description}"></i>
+                        <#if !(reportModel.projectModel??) && reportModel.isExportAllIssuesCSV>
+                            <a class="csvReport" style="font-size: 14pt;" align="right" href="../AllIssues.csv">(Download All Issues CSV)</a>
```

Comment:
```suggestion
                            <span style="float: right;">
                            <a class="csvReport" style="font-size: 14pt;" align="right" href="../AllIssues.csv">Download All Issues CSV</a>
                            </span>
```

Reply:
Ah, sorry, I missed that, thanks. I'll check this in.